specs/Spec_for_adding_user.adoc

```diff
@@ -0,0 +1,166 @@
+# User Management: Adding new User in Tendrl
+
+## Problem description
+
+This spec talks about Adding/Creating a new User to Tendrl by the Admin
+
+## Use Cases
+
+The flow goes as follows:-
+
+* A user login as Admin
+* Clicks on Admin tab in Navigation
+* The navigation drops down
+* Clicks on *Users*
+* Clicks on *Add* button in Users page view
+
+User will land onto https://redhat.invisionapp.com/share/KNB25OEQT#/screens/226063802[Add Users] page
+
+## Proposed change
+
+### UI changes
+
+* Add *Users* tab in Admin navigation tab
+* Add *Add User* Button in Users Page view
+* Add a form for adding a user. The form will contain following feilds:-
+** User ID
+*** Input type : text
+*** Validations:
+**** Should not contain special characters
```

Comment:
@brainfunked @anivargi @nthomas-redhat What validations are required for User ID field? If we want to check for uniqueness then we will require an API call to do that. What should be the minimum and maximum length allowed for this field.